### DISCUSSION TITLE
Trying to install Budibase in Ubuntu version 20.

### DISCUSSION THREAD

**Discussion by hohemathaus** [Budibase/budibase#2047] [General]
Hi guys, kindly assist as I am experiencing error each time I try to install budibase. The error is as shown below.
Get https://registry-1.docker.io/v2/: dial tcp: lookup registry-1.docker.io: Temporary failure in name resolution.
Budibase Error
Creating network "budibase_default" with the default driver
Creating volume "budibase_couchdb3_data" with local driver
Creating volume "budibase_minio_data" with local driver
Creating volume "budibase_redis_data" with local driver
Pulling minio-service (minio/minio:)...
Get https://registry-1.docker.io/v2/: dial tcp: lookup registry-1.docker.io: Temporary failure in name resolution

**Comment by KernOil** [comment]
I would look at your docker install I think that may be where the error is coming from. make sure you have docker and docker-compose installed properly

**Reply by mike12345567** [reply]
Hi @hohemathaus - it looks like your setup is having issues resolving the docker images, as @AndrewJRamay says this is likely a problem with your machine/docker configuration.

Looked online and here is someone having a very similar problem: https://stackoverflow.com/questions/47580528/error-response-from-daemon-get-https-registry-1-docker-io-v2-dial-tcp-look
